src/coprocessor/statistics/analyze.rs

```diff
@@ -51,6 +51,7 @@ pub struct AnalyzeContext<S: Snapshot> {
     ranges: Vec<KeyRange>,
     storage_stats: Statistics,
     quota_limiter: Arc<QuotaLimiter>,
+    is_auto: bool,
```

Comment:
```suggestion
    background: bool,
```

Reply:
> 

renamed it to more precise of is_auto_analyze.

Reply:
No, the point is to use `background`. `is_auto_analyze` is a concept only meaningful to rate limiter, but `AnalyzeContext` is a representation of task.

Reply:
> > 
> 
> renamed it to more precise of is_auto_analyze.

is_auto_analyze is a natural attribute for analyze context than background. Only rate limiter will treat it as background.

Reply:
Oh right. I just misread it as `is_auto_tune`...